src/Polly.Shared/PolicyAsync.cs

```diff
@@ -28,10 +28,21 @@ internal Policy(Func<Func<Task>, Task> asyncExceptionPolicy, IEnumerable<Excepti
         [DebuggerStepThrough]
         public Task ExecuteAsync(Func<Task> action)
         {
+            return ExecuteAsync(false, action);
+        }
+
+        /// <summary>
+        ///     Executes the specified asynchronous action within the policy.
+        /// </summary>
+        /// <param name="continueOnCapturedContext">Whether to continue on a captured synchronization context.</param>
+        /// <param name="action">The action to perform.</param>
+        [DebuggerStepThrough]
+        public Task ExecuteAsync(bool continueOnCapturedContext, Func<Task> action)
```

Comment:
Request move `bool continueOnCapturedContext` to be last parameter, in `ExecuteAsync()`